### ISSUE TITLE
Debt: Refactor deployments endpoint

### ISSUE THREAD

**Issue by willcbelcher** [rropen/SFM#45]
**Topic**
Deployments endpoint needs to be updated. The deployment_dates key needs to contain an array, where each entry in the array is another array of length 2, containing data as follows: [unix datetime for that day, number of successful deployments for that day]. All days need to be included in this list, even if they have no deployments. 

**Service**
Backend

**Steps**
- [ ] Update endpoint with new data format
- [ ] Update all endpoints to use camelCase in their fields (e.g. deploymentFrequency) rather than snake_case as now (e.g. deployment_frequency) 
- [ ] Add description of possible deployment frequency ratings to the endpoint

**Comment by jdhaines** [comment]
@willcbelcher add an example short array in the issue using code in the markdown... Like:

```json
{
  [
    {
      "time": 1193849403922,
      "val": 3
    }
  ]
}
```

That way they don't have to guess.

**Comment by willcbelcher** [comment]
Example endpoint:

`
[
  {
    "project_name": "org",
    "deployment_dates": [
      [ 1625184000, 0],
      [ 1625788800.0,, 1]
    ],
    "deployment_frequency": "Daily"
  }
]
`

**Comment by jdhaines** [comment]
Typed the wrong issue number in another commit.

**Comment by jdhaines** [comment]
Added another task to this issue.

**Comment by willcbelcher** [comment]
Added another task to this issue

**Comment by TylerB890** [comment]
closed and passing tests via d081eaa
